.solhint.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
Why are we removing solhint? Does hardhat have linter?

Reply:
It complains too much about our own sytle of using double undescores for internal and immutable fields